core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -37,41 +37,6 @@
 import org.apache.carbondata.core.metadata.ColumnarFormatVersion;
 import org.apache.carbondata.core.util.annotations.CarbonProperty;
 
-import static org.apache.carbondata.core.constants.CarbonCommonConstants.BLOCKLET_SIZE;
```

Comment:
Why this change is required ??

Reply:
For example, when inside the function, BLOCKLET_SIZE is used, then CarbonCommonConstants.BLOCKLET_SIZE is more readable then just BLOCKLET_SIZE.